classes/Renderer/Footer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace MySociety\TheyWorkForYou\Renderer;
+
+/**
+ * Template Footer Renderer
+ *
+ * Prepares variables for inclusion in a template, as well as handling variables
+ * for use in header and footer.
+ */
+
+class Footer
+{
+
+    /**
+     * Output Page
+     *
+     * Assembles the data required to display the page footer
+     *
+     * @param array  $data     An associative array of data to be made available to the template.
+     */
+
+    public function get_data($data = array())
+    {
+        global $DATA, $this_page, $THEUSER;
+
+        $data['footer_links']['about'] = self::get_menu_links(array ('help', 'about', 'linktous', 'houserules', 'blog', 'news', 'contact', 'privacy'));
+        $data['footer_links']['assemblies'] = self::get_menu_links(array ('hansard', 'sp_home', 'ni_home', 'wales_home', 'boundaries'));
+        $data['footer_links']['international'] = self::get_menu_links(array ('newzealand', 'australia', 'ireland', 'mzalendo'));
+        $data['footer_links']['tech'] = self::get_menu_links(array ('code', 'api', 'data', 'pombola', 'devmailinglist', 'irc'));
```

Comment:
Rather than $data being passed in to this and passed back, should get_data just return an array that the parent can assign to data['footer_links'] or similar?